docs/compatibility.md

```diff
@@ -0,0 +1,7 @@
+## Compatibility
+
+PyJelly is tested and supported with the following dependency versions.
+
+| Component | Supported versions |
+|----------|---------------------|
+| RDFLib   | **7.x** (>= 7.1.4)  |
```

Comment:
Missing: Python version, pyjelly version. Follow the format of the table we have for Jelly-JVM.